app/http.php

```diff
@@ -490,7 +490,7 @@ function cleanUp(Orchestration $orchestration, Table $activeRuntimes, array $net
     ->action(function (string $runtimeId, string $image, string $entrypoint, string $source, string $destination, string $outputDirectory, array $variables, string $runtimeEntrypoint, string $command, int $timeout, bool $remove, float $cpus, int $memory, string $version, string $restartPolicy, array $networks, Orchestration $orchestration, Table $activeRuntimes, Response $response, Log $log) {
         $runtimeName = System::getHostname() . '-' . $runtimeId;
 
-        $runtimeHostname = \uniqid();
+        $runtimeHostname = \bin2hex(\random_bytes(16));
```

Comment:
Lets include both timestamp-based ID, And randomness

Reply:
This is using 16 bytes * 8 bits/byte = 128 bits of entropy
With 128 bits of entropy, the chance of a collision is negligible, even if we're generating billions of hostnames.
You'd need  18 quintillion hostnames before the probability of a collision becomes significant.